### DISCUSSION TITLE
Support EPUB conversion?

### DISCUSSION THREAD

**Discussion by baek-sang** [Stirling-Tools/Stirling-PDF#1558] [Q&A] [answered]
epub -> pdf,
pdf -> epub

Can I convert?

**Comment by Frooodle** [comment, marked as answer]
yes (in docker)
set env variable ``INSTALL_BOOK_AND_ADVANCED_HTML_OPS`` to true
And you can use either latest or latest-fat , for your case I recommend  latest-fat
